controllers/storagecluster/provider_server.go

```diff
@@ -42,6 +42,7 @@ const (
 
 	ocsClientConfigMapName = "ocs-client-operator-config"
 	deployCSIKey           = "DEPLOY_CSI"
+	manageNoobaaSub        = "MANAGE_NOOBAA_SUB"
```

Comment:
Why do we use all caps + snake case for configmap keys?

Reply:
was following the same pattern that was used for CSI above, it can be change imo.